util/utils.go

```diff
@@ -20,13 +20,11 @@ type Change struct {
 //Schedule goroutine that will ping other goroutines via send channel
 func Schedule(send chan bool, duration string) {
 	u, err := time.ParseDuration(duration)
-	if err == nil {
-		for {
-			send <- true
-			time.Sleep(u)
-		}
-	} else {
-		fmt.Println("Couldn't set a scheduler due to improper time format: " + duration)
+	if err != nil {
+		log.Fatalf("Couldn't set a scheduler due to improper time format: %s\n", duration)
```

Comment:
You know `log.Fatalf` stops the program right?

Reply:
Well, I actually didn't know it ! Despite the fact that this software is written in Go I don't write that much Go 😸. You can disregard my previous post. Sorry for not checking it out, it just didn't come to my mind that logging statement could terminate program as well!